code/modules/cargo/bounties/special.dm

```diff
@@ -26,6 +26,13 @@
 	required_count = 10
 	wanted_types = list(/obj/item/stack/sheet/mineral/adamantine)
 
+/datum/bounty/item/trash
+	name = "Trash"
+	description = "A set of janitors who are soon going to be out of jobs are requesting for some trash to be sent to them to proove themselves worthy to centcomm."
```

Comment:
Grammar

Reply:
Points out which in specific, i'm not exactly an english speaker here.

Reply:
proove / prove, centcomm / Centcom
Btw I think you should rephrase it like "Some janitors, who are soon going to run out of thrash to clean, are requesting thrash so that they can keep proving themselves worthy to Centcom."

Reply:
In the changed text there's no space after the `,`. Also you may want to resolve this review after you're done.